core/src/main/scala/org/apache/spark/h2o/backends/internal/InternalBackendConf.scala

```diff
@@ -41,6 +41,9 @@ trait InternalBackendConf extends SharedBackendConf {
   def nodeIcedDir = sparkConf.getOption(PROP_NODE_ICED_DIR._1)
   def nodeNetworkMask = sparkConf.getOption(PROP_NODE_NETWORK_MASK._1)
 
+  def isInternalSecureConnectionsEnabled = sparkConf.getBoolean(PROP_INTERNAL_SECURE_CONNECTIONS._1,
```

Comment:
Corresponding method `isInternalSecureConnectionsDisabled` is missing. All methods working with booleans have these 2 methods in the current api, so it's good for consistency